### PR TITLE
Add eslint integration for react, fix all files

### PR DESCRIPTION
**Pull request by ahmedsakr**
* I integrated eslint into the project with the configuration file living in `.eslintrc.json`.
* GitHub Job that automatically runs the lint to check compliance

### CODE REVIEW

File: components/Footer.js [hunk 1]
```diff
@@ -1,43 +1,45 @@
...
-        alignItems: 'center'
-    }
-});
\ No newline at end of file
+  footer: {
+    height: 75,
+    backgroundColor: '#1D1D1D',
+    width: '100%',
+    flexDirection: 'row',
+    justifyContent: 'space-around',
```
Comment (MichaelCalabro) [inline components/Footer.js:35]: Why commas on the last item? Can this be disabled? If not thats ok

Comment (ahmedsakr) [reply components/Footer.js:35]: It's common practice to leave a comma on the last one so if you add more later, you don't have to modify this line

**Review by joshcampitelli — COMMENTED** [review]
`.eslintrc.json` has 4 space indentation, yet other `.json` files are 2 - Besides that looks good 🐌

File: components/Footer.js (hunk 1 above)
Comment (MichaelCalabro) [reply components/Footer.js:35]: Oh ok

**Review by MichaelCalabro — APPROVED** [review]

**Review by joshcampitelli — APPROVED** [review]
LGTM :shipit:

**Pull request merged**